app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -444,6 +447,19 @@ def mail_delivery_params(proofed_at)
     return { wait_until: wait_until, queue: :intentionally_delayed }
   end
 
+  def send_enrollment_status_sms_notification(enrollment:)
+    return unless IdentityConfig.store.in_person_send_proofing_notifications_enabled
+    return if enrollment&.proofed_at.blank?
+    sms_delay_hours = IdentityConfig.store.in_person_results_delay_in_hours ||
+                      DEFAULT_EMAIL_DELAY_IN_HOURS
+    wait_until = enrollment.proofed_at + sms_delay_hours
+    if wait_until <= Time.zone.now
+      InPerson::SendProofingNotificationJob.set(wait_until: wait_until).perform_later(enrollment.id)
+    else
+      InPerson::SendProofingNotificationJob.perform_now(enrollment.id)
+    end
```

Comment:
I don't think we should use `perform_now` here, even if we don't want to wait before sending the SMS message. My understanding is that `perform_now` will block this thread of execution until the called job completes. I think it would be better to enqueue it with `perform_later` without any `wait_until` option and it will be executed as a separate job

Reply:
@sheldon-b , good point, so i assume perform_later works without a wait. Fixed.